packages/ott-vis-panel/src/components/ForceGraph.tsx

```diff
@@ -140,6 +140,29 @@ const ForceGraph: React.FC<ForceGraphProps> = ({
 		return num * 2;
 	}
 
+	let numRooms = nodes.filter(node => node.group === "room").length;
+
+	if (numRooms > 50) {
+		let consolidatedNodes = [];
+		let roomNodes = nodes.filter(node => node.group === "room");
+		let otherNodes = nodes.filter(node => node.group !== "room");
+
+		for (let i = 0; i < roomNodes.length; i += 2) {
+			if (roomNodes[i + 1]) {
+				consolidatedNodes.push({
+					...roomNodes[i],
+					radius: roomNodes[i].radius * 2,
+					text: roomNodes[i].text + ", " + roomNodes[i + 1].text,
+				});
+			} else {
+				consolidatedNodes.push(roomNodes[i]);
+			}
+		}
+
+		nodes = [...consolidatedNodes, ...otherNodes];
+		numRooms = nodes.filter(node => node.group === "room").length;
+	}
```

Comment:
I think this would be way easier to do if you did it in the `View`s

Reply:
Ah yea, probably. Taking a look now. Are we only having this problem in `GlobalView` or both of em?

Reply:
Yeah just do it in the global view for now